src/app/backing/BackingPage.tsx

```diff
@@ -19,6 +28,19 @@ export const BackingPage = () => {
   const { address } = useAccount()
   const { data: rewardsData, error: rewardsError } = useGetBuildersRewards(getTokens())
   useHandleErrors({ error: rewardsError, title: 'Error loading builder rewards' })
+  const { prices } = usePricesContext()
+  // TODO: add useAllocationsContext hook?
+  const { state } = useContext(AllocationsContext)
+  const votingPower = state.backer.balance
+  const totalOnchainAllocation = state.backer.amountToAllocate
+  const rifPriceUsd = prices[RIF]?.price ?? 0
+
+  const availableForBacking =
+    !votingPower || !totalOnchainAllocation ? 0 : Number(votingPower - totalOnchainAllocation)
+
+  const totalBacking = !totalOnchainAllocation ? 0 : Number(totalOnchainAllocation)
```

Comment:
Not sure if casting this value is ok, as we may loose precision. let's replicate what we do in other cases